templates/templates/node--16.html.twig

```diff
@@ -42,7 +42,7 @@
               'alert_text': "This page is currently unpublished and not visible to the public."|t,
             }
           %}
-          {% include "@basic-dist/decanter/components/alert/alert.twig" with data %}
+          {% include "@decanter/components/alert/alert.twig" with data %}
```

Comment:
Updating this since Jacob kindly informed me that there were changes made on how decanter is imported in this [PR](https://github.com/SU-SWS/stanford_profile/commit/b0f7b0c243aef5ff5ab6df48d5fef8b0cb5a5f8a) 😄 
(Thank you @jdwjdwjdw!)